### DISCUSSION TITLE
HttpRequestException: 403 (Forbidden)

### DISCUSSION THREAD

**Discussion by sdomenici009** [Tyrrrz/YoutubeExplode#816] [Q&A]
Has anyone else hit this recently? Just started seeing it in the past day or two on every video. Tried updating to the latest version but same issue.

**Comment by aarushpatilt** [comment]
same issue here unsure how to fix

**Reply by aarushpatilt** [reply]
im using flutter

**Reply by aarushpatilt** [reply]
running into same issue when using ytdl, geeting 403 or 410, might be a youtube security update. not sure.

**Reply by cata76** [reply]
Me too.
All links give 403 error (Forbidden)

**Reply by highwingers** [reply]
let me know if you find a workaround.

**Reply by EBTYX2809** [reply]
Guys, new version fix it.

**Reply by jtwhitehd** [reply]
[The new version release](https://github.com/Tyrrrz/YoutubeExplode/releases/tag/6.4.2) has a disclaimer that says, "⚠️ Due to the recent changes by YouTube, muxed streams are no longer provided. Please make sure to use audio-only and video-only streams and combine them as needed."

**Reply by klaxxon123** [reply]
muxed streams are no longer provided (parsed). try to get them via some silent webview2 and parse it via some executescriptasync or something :). Just suggestion.
